Feature/deploy to nuget

stefaneberl opened the pull request.

stefaneberl committed:
Add files required for packaging

stefaneberl committed:
Finalizing appveyor for testing

y0chanan commented on packaging/OpenKit.NET.package.ps1:
```diff
@@ -0,0 +1,114 @@
+﻿# Powershell script to package OpenKit .NET into a NuGet package
```
as discussed: instead of relative addressing '..\' use pwd as base folder

stefaneberl committed:
Use $PSScriptRoot to determine location of script

All paths were assuming that the script was executed in the
same directory where it resides.

This has been changed by prepending $PSScriptRoot in front of relative
paths.
